src/python/pants/backend/jvm/tasks/jvm_compile/rsc/rsc_compile.py

```diff
@@ -498,14 +503,19 @@ def nonhermetic_digest_classpath():
               youtline_args = wartremover_args + youtline_args
 
           target_sources = ctx.sources
-          args = [
-                   '-cp', os.pathsep.join(classpath_entry_paths),
-                   '-d', rsc_jar_file_relative_path,
-                 ] + self.get_options().extra_rsc_args + youtline_args + target_sources
+          
+          # TODO: m.jar digests aren't found, so hermetic will fail.
+          if use_youtline and not hermetic and self.get_options().zinc_outline:
+            self._zinc_outline(ctx, classpath_paths, target_sources, youtline_args)
```

Comment:
`hermetic` would not be a factor here, would it?

Reply:
I'm actually getting a digest not found error for some m.jar. Not sure it's worth pursuing in this PR.